pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
Sorry, I can't seem to find the official go documentation of this feature, but here is a decent explanation, scroll down to "black-box test package" https://splice.com/blog/lesser-known-features-go-test/